CertificateTroubleshoot: add openssl client support (#10632)
demisto/content: Packs/Troubleshoot/Scripts/CertificatesTroubleshoot/README.md

After following the [tutorial](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/docker/configure-python-docker-integrations-to-trust-custom-certificates) to update your custom certificate in `Cortex XSOAR Server`/ `Cortex XSOAR Engine`, validate the configuration applied using this script.

> The Script supports two modes of operation:

Reviewer: ```suggestion
The script supports two modes of operation:
```

>   1. **python**: will use the python built-in SSL library to detect the endpoint's certificates.

Reviewer: ```suggestion
  1. **python**: Uses the Python built-in SSL library to detect the endpoint's certificates.
```

>   2. **openssl**: will use the openssl client to detect the endpoint's certificates. Use this mode if the `python` mode fails for some reason.

Reviewer: ```suggestion
  2. **openssl**: Uses the OpenSSL client to detect the endpoint's certificates. Use this mode if the `python` mode fails for some reason.
```

| --- | --- |
| endpoint | The endpoint identifier IP address or URL:Port. If the port is not included, 443 will be used by default. |
| port | The endpoint port. Default is 443. |

> | mode | Operation mode. Determines how the endpoint is inspected. Either using python built-in SSL or openssl client. |

Reviewer: ```suggestion
| mode | Operation mode. Determines how the endpoint is inspected. Either using Python built-in SSL or OpenSSL client. |
```
